lib/ui/table.ts

```diff
@@ -1,12 +1,16 @@
-// @ts-nocheck
-const chalk = require('chalk');
-const { table } = require('table');
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Could we disable this only in the spots needed instead of the entire file? 🙏

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1233/commits/f413ece568f46737096595ea5ea01c905f53d936